### PR TITLE
New design

### PR DESCRIPTION
**Pull request by pdt1806**
new design cuz eggu learned (watched a yt video for a few minutes) how to design better pages in figma and used this website as an albino mouse siuuuuuuu

jk this is called evolution

### CODE REVIEW

File: package-lock.json
Comment (code-ga) [inline package-lock.json:1]: is this file necessity

File: src/components/Footer/index.tsx
Comment (code-ga) [inline src/components/Footer/index.tsx]: try another name ?

File: src/pages/Docs/MainMenu/Support/Still/index.tsx
Comment (code-ga) [inline src/pages/Docs/MainMenu/Support/Still/index.tsx]: why don't we move this outside function

File: src/pages/Home/Features/index.tsx
Comment (code-ga) [inline src/pages/Home/Features/index.tsx]: move outside function?

File: src/pages/Home/Offers/index.tsx
Comment (code-ga) [inline src/pages/Home/Offers/index.tsx]: move outside function?

File: src/pages/Home/Testimonials/index.tsx
Comment (code-ga) [inline src/pages/Home/Testimonials/index.tsx]: try another sentence

File: src/pages/Home/Testimonials/index.tsx
Comment (code-ga) [inline src/pages/Home/Testimonials/index.tsx]: i think we should query this in database

File: src/pages/Pricing/Options/index.tsx
Comment (code-ga) [inline src/pages/Pricing/Options/index.tsx]: rename some variable pls

**Review by code-ga — CHANGES_REQUESTED** [review]
code style review, unreviewed code logic

**Comment by code-ga** [comment]
> new design cuz eggu learned (watched a yt video for a few minutes) how to design better pages in figma and used this website as an albino mouse siuuuuuuu
> 
> jk this is called evolution

nice

File: src/pages/Home/Testimonials/index.tsx
Comment (tobycm) [reply src/pages/Home/Testimonials/index.tsx]: Nahh

File: package-lock.json
Comment (tobycm) [reply package-lock.json:1]: Yes

File: src/App.tsx
Comment (tobycm) [inline src/App.tsx]: Dont do a double lookup, i will refactor a lil bit for u later

File: src/App.tsx
Comment (tobycm) [inline src/App.tsx:23]: Also why dont we bring this outside the component?

File: src/components/FeaturesCard/index.tsx
Comment (tobycm) [inline src/components/FeaturesCard/index.tsx]: Just call props feature at this point:)))

File: src/components/Footer/index.tsx
Comment (tobycm) [reply src/components/Footer/index.tsx]: I should take a look at this later with VSCode

File: src/components/OffersNumber/index.tsx
Comment (tobycm) [inline src/components/OffersNumber/index.tsx]: Same thing here, just call props as offer then you can do stuff like `<OffersNumber number={69} title="God" />`

File: src/components/PricingBox/index.tsx
Comment (tobycm) [inline src/components/PricingBox/index.tsx]: Bring these outside of the component :)

File: src/components/TestimonialBox/index.tsx
Comment (tobycm) [inline src/components/TestimonialBox/index.tsx]: Same as before

File: src/database/index.ts
Comment (tobycm) [inline src/database/index.ts]: I would write this in 1 line 😁

File: src/pages/Home/Testimonials/index.tsx
Comment (tobycm) [inline src/pages/Home/Testimonials/index.tsx]: Eggu u should add me in here and pick random 3 to show

File: src/pages/Pricing/Breakdown/index.tsx
Comment (tobycm) [inline src/pages/Pricing/Breakdown/index.tsx]: Outside too

File: src/pages/Pricing/Options/index.tsx
Comment (tobycm) [inline src/pages/Pricing/Options/index.tsx]: Add type too, feature array element is quite ambiguous

File: src/pages/Pricing/Questions/index.tsx
Comment (tobycm) [inline src/pages/Pricing/Questions/index.tsx]: Ctrl K Ctrl L Ctrl L Ctrl L Ctrl X ⬆️ ⬆️ Ctrl V

**Review by tobycm — CHANGES_REQUESTED** [review]
Very nice work eggu

File: src/App.tsx
Comment (code-ga) [reply src/App.tsx:23]: i think so

**Comment by code-ga** [comment]
egg after seen request change:

**Comment by pdt1806** [comment]
eggu unintentionally created the most sôi nổi PR

**Comment by pdt1806** [comment]
btw i will resolve some today

**Comment by code-ga** [comment]
> eggu unintentionally created the most sôi nổi PR

review from vn and cad

File: package-lock.json
Comment (code-ga) [reply package-lock.json:1]: resolve this now

File: src/components/Footer/index.tsx
Comment (pdt1806) [reply src/components/Footer/index.tsx]: i have no idea how to name the list

File: src/components/OffersNumber/index.tsx
Comment (pdt1806) [reply src/components/OffersNumber/index.tsx]: yea i was thinking of doing like that, but then for some reasons i dropped that idea

File: src/App.tsx
Comment (pdt1806) [reply src/App.tsx:23]: > Also why dont we bring this outside the component?

react hooks can only be used inside a component function

File: src/pages/Home/Testimonials/index.tsx
Comment (pdt1806) [reply src/pages/Home/Testimonials/index.tsx]: actually i would pick 3 testers (connect be to fe fast pls)

File: src/App.tsx
Comment (code-ga) [reply src/App.tsx:23]: > > Also why dont we bring this outside the component?
> 
> react hooks can only be used inside a component function

I think he mean the routes variable

File: src/components/Footer/index.tsx
Comment (code-ga) [reply src/components/Footer/index.tsx]: > i have no idea how to name the list

try to rename that to make that more readable pls

File: src/App.tsx
Comment (pdt1806) [reply src/App.tsx:23]: > > > Also why dont we bring this outside the component?
> > 
> > 
> > react hooks can only be used inside a component function
> 
> I think he mean the routes variable

there is a hook call inside that routes list

Comment (code-ga) [reply src/App.tsx:23]: > > > > Also why dont we bring this outside the component?
> > > 
> > > 
> > > react hooks can only be used inside a component function
> > 
> > 
> > I think he mean the routes variable
> 
> there is a hook call inside that routes list

hmm I think that we can make this resolve but just wait for @tobycm's reply

**Comment by code-ga** [comment]
> new design cuz eggu learned (watched a yt video for a few minutes) how to design better pages in figma and used this website as an albino mouse siuuuuuuu
> 
> jk this is called evolution

refactor some code and I can review it again tomorrow =>> or some day later

File: src/pages/Home/Testimonials/index.tsx
Comment (code-ga) [reply src/pages/Home/Testimonials/index.tsx]: > actually i would pick 3 testers (connect be to fe fast pls)

backlog belike : come here

**Comment by pdt1806** [comment]
got ya

**Comment by code-ga** [comment]
thanks

File: src/pages/Home/Testimonials/index.tsx
Comment (pdt1806) [reply src/pages/Home/Testimonials/index.tsx]: riel

File: src/App.tsx
Comment (tobycm) [reply src/App.tsx:23]: > > Also why dont we bring this outside the component?
> 
> react hooks can only be used inside a component function

ah ok i see

File: src/database/index.ts
Comment (tobycm) [reply src/database/index.ts]: @pdt1806 bro i personally would but u shouldn't
because almost 0 readability xddddd

File: src/pages/Home/Testimonials/index.tsx
Comment (tobycm) [reply src/pages/Home/Testimonials/index.tsx]: @pdt1806 okey make a place for testers to put their comment on fe to be

**Comment by tobycm** [comment]
@CookieGMVN cháy email luôn

File: src/components/Footer/index.tsx
Comment (pdt1806) [reply src/components/Footer/index.tsx]: > > i have no idea how to name the list
> 
> try to rename that to make that more readable pls

change from `allData` to `contentData` cuz each (list of) data is used locally for that file only (that is, the content of the component only) so i just call it with a general name like that

Comment (tobycm) [reply src/components/Footer/index.tsx]: broken english boi demanding different variable name from boi living in US

File: src/components/DocumentSectionBox/index.tsx
Comment (tobycm) [inline src/components/DocumentSectionBox/index.tsx]: erm do `document` instead of `{
  section,
  title,
  icon: IconComponent,
  documents
}` :))

Comment (tobycm) [reply src/components/DocumentSectionBox/index.tsx]: ur choice tho

File: src/components/FeaturesCard/index.tsx
Comment (tobycm) [inline src/components/FeaturesCard/index.tsx]: same thing here

File: src/components/Footer/index.tsx
Comment (tobycm) [inline src/components/Footer/index.tsx]: Might as well pull the type into a new type variable

File: src/components/OffersNumber/index.tsx
Comment (tobycm) [inline src/components/OffersNumber/index.tsx]: same thing

File: src/components/PricingBox/index.tsx
Comment (tobycm) [inline src/components/PricingBox/index.tsx]: +

File: src/pages/Docs/documents_list.ts
Comment (tobycm) [inline src/pages/Docs/documents_list.ts]: pull type out too

File: src/pages/Home/Features/index.tsx
Comment (tobycm) [inline src/pages/Home/Features/index.tsx]: pull type out too

File: src/pages/Home/Offers/index.tsx
Comment (tobycm) [inline src/pages/Home/Offers/index.tsx]: very creative property name

File: src/pages/Home/Testimonials/index.tsx
Comment (tobycm) [inline src/pages/Home/Testimonials/index.tsx:20]: ai cho nghi ngờ

anyway để cái này permanent luôn nha

File: src/pages/Pricing/Questions/index.tsx
Comment (tobycm) [inline src/pages/Pricing/Questions/index.tsx]: what is `value`?

File: src/components/DocumentSectionBox/index.tsx
Comment (pdt1806) [reply src/components/DocumentSectionBox/index.tsx]: oh bruh i thought u wanted to split into multiple props

Comment (pdt1806) [reply src/components/DocumentSectionBox/index.tsx]: whatever i've been doing like this for other react projects

File: src/components/Footer/index.tsx
Comment (pdt1806) [reply src/components/Footer/index.tsx]: hmm ok

File: src/pages/Pricing/Questions/index.tsx
Comment (pdt1806) [reply src/pages/Pricing/Questions/index.tsx]: basically to help manage accordion state
[read more](https://mantine.dev/core/accordion)

**Comment by pdt1806** [comment]
u guys can help me with code readability at this point

File: src/pages/Home/Testimonials/index.tsx
Comment (code-ga) [reply src/pages/Home/Testimonials/index.tsx]: put that to the backlog

File: src/pages/Docs/Document/index.tsx
Comment (code-ga) [inline src/pages/Docs/Document/index.tsx:1]: some rewrite this to be more beautiful ( mark as resolved if no one can do that )

File: src/pages/Home/HeroHeader/index.tsx
Comment (code-ga) [inline src/pages/Home/HeroHeader/index.tsx]: wtf why use clearTimeout for setInterval

File: src/pages/Home/Testimonials/index.tsx
Comment (code-ga) [reply src/pages/Home/Testimonials/index.tsx:20]: vãi

**Review by code-ga — CHANGES_REQUESTED** [review]
i lost review a logical bug

File: src/pages/Home/HeroHeader/index.tsx
Comment (code-ga) [reply src/pages/Home/HeroHeader/index.tsx]: @pdt1806

**Comment by code-ga** [comment]
> u guys can help me with code readability at this point

sorry bro I only can review because I am so busy learning for the exam

File: src/pages/Home/Testimonials/index.tsx
Comment (code-ga) [reply src/pages/Home/Testimonials/index.tsx:20]: image

**Review by code-ga — APPROVED** [review]
nice work

**Review by tobycm — APPROVED** [review]
🛌

**Comment by code-ga** [comment]
@pdt1806 merge đi bro

**Comment by code-ga** [comment]
ko bỏ lỡ một bug nào đâu đúng ko

**Pull request merged**
